taqueria-plugin-tezos-client/README.md

```diff
@@ -1,15 +1,15 @@
 # Taqueria Tezos Client Plugin
 
-The Tezos Client plugin provides tasks to analyze contracts including type checking and simulating of Michelson (`.tz`) smart contracts. They both execute against a protocol (currently jakarta). We'll enable a `--protocol` flag in the future to allow users to specify a protocol to use
+The Tezos Client plugin provides tasks to analyze contracts including type checking and simulating of Michelson (`.tz`) smart contracts. They both execute against a protocol (currently kathmandu). We'll enable a `--protocol` flag in the future to allow users to specify a protocol to use
```

Comment:
When we refactor environments, we can make protocol a field of the environment object. I think that could then be used as the default protocol.